src/validation_workflow/deb/validation_deb.py

```diff
@@ -0,0 +1,60 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import os
+import time
+
+from system.execute import execute
+from test_workflow.integ_test.utils import get_password
+from validation_workflow.api_test_cases import ApiTestCases
+from validation_workflow.download_utils import DownloadUtils
+from validation_workflow.validation import Validation
+from validation_workflow.validation_args import ValidationArgs
+
+
+class ValidateDeb(Validation, DownloadUtils):
+    def __init__(self, args: ValidationArgs) -> None:
+        super().__init__(args)
```

Comment:
Missing tests.

Reply:
I'm working on the test cases. Will update it in next commit. Thanks